src/theme/foundations/spacing.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * This is based on:
+ *   root size of 16px or 1rem
+ *   1 spacing unit is 4px or 0.25rem
+ *
+ * Chakra's mental model: If you need a spacing of 40px, divide it by 4.
+ * That'll give you 10. Then use it in your component.
+ *
+ * The DS uses eight CSS variables/values found in "src/styles/03-space/_space.css":
+ *   --space-xxs = 4px or 0.25rem
+ *   --space-xs = 8px or 0.5rem
+ *   --space-s = 16px or 1rem
+ *   --space-m = 24px or 1.5rem
+ *   --space-l = 32px or 2rem
+ *   --space-xl = 48px or 3rem
+ *   --space-xxl = 64px or 4rem
+ *   --space-xxxl = 96px or 6rem
+ *
+ * @note Even though all the following values are available through Chakra,
+ * we recommend to use the spacing values that map to the DS values
+ * declared above.
+ *
+ * Chakra Value | DS Variable
+ * ----------------------------
+ *     1        |  --space-xxs
+ *     2        |  --space-xs
+ *     4        |  --space-s
+ *     6        |  --space-m
+ *     8        |  --space-l
+ *     12       |  --space-xl
+ *     16       |  --space-xxl
+ *     24       |  --space-xxl
+ */
+export const spacing = {
```

Comment:
For typography, "0" aligns with "1rem," which is our base.  What about simplifying the spacing object similarly and using an array of consecutive numbers to match up with the origina NYPL DS spacing values?  Also, can we use negative numbers for the key names?

Reply:
This spacing file is not for typography-related style rules. Check out the `typography.ts` file where it does have `"0": "1rem"` and it also does include negative font-size values.

Reply:
I see what you're saying. I took those values from the existing `.css` files which do match up nicely with Chakra's default values. I think it is technically possible to update the spacing values to be aligned with the typography values and its convention but it seems like it starts on different values on purpose.